automobile/cpp-libraries.md

```diff
@@ -15,6 +15,7 @@ The `Driver` class provides the following methods:
 | class [Driver](driver-library.md) {                                                                          |
 | &nbsp;&nbsp; enum {INDICATOR\_OFF, INDICATOR\_RIGHT, INDICATOR\_LEFT};                                       |
 | &nbsp;&nbsp; enum {SPEED, TORQUE };                                                                          |
+| &nbsp;&nbsp; enum {WIPER\_DOWN, WIPER\_SLOW, WIPER\_NORMAL, WIPER\_FAST};                                    |
```

Comment:
This `WIPER` prefix seems wrong (not present in https://github.com/omichel/webots/blob/63f1a00748563bb79255b9a061e757288c0ef8dc/projects/automobile/libraries/CppDriver/include/webots/Driver.hpp#L25)